make-webpack-config.js

```diff
@@ -170,27 +175,37 @@ module.exports = function(options) {
     ];
 
 
-    // The following variable are about how the frame lib will be referenced in the iframe html.
-    // see `FRAME_LIB_URL` in `src/host/js/smooch.js`.
-    // In host and npm mode, it's referencing a file that should already (or soon to) be on the CDN
+    // The following variables are about how the frame files will be referenced in the iframe html.
+    // see `FRAME_JS_URL` and `FRAME_CSS_URL` in `src/host/js/smooch.js`.
+    // In host and npm mode, it's referencing files that should already (or soon to) be on the CDN
     // so it should target the full name + version.
-    // In other cases, iframe.js will do just fine.
-    let frameLibFilename;
+    // In other cases, iframe.js/css will do just fine.
+    let frameJsFilename;
+    let frameCssFilename;
 
     if (['host', 'npm'].includes(buildType)) {
         // in this case, it's referencing an already built frame lib
         // and it's mostly likely minified already.
-        frameLibFilename = `frame.${VERSION}.min.js`;
+        frameJsFilename = `frame.${VERSION}.min.js`;
+        frameCssFilename = `frame.${VERSION}.css`;
     } else {
-        frameLibFilename = 'frame.js';
+        frameJsFilename = 'frame.js';
+        frameCssFilename = 'frame.css';
     }
 
     const plugins = [
         new webpack.DefinePlugin({
-            FRAME_LIB_URL: `'${publicPath}${frameLibFilename}'`
+            FRAME_JS_URL: `'${publicPath}${frameJsFilename}'`,
+            FRAME_CSS_URL: `'${publicPath}${frameCssFilename}'`
         })
     ];
 
+    if (buildType === 'frame') {
+        plugins.push(new ExtractTextPlugin(`frame.${VERSION}.css`));
```

Comment:
you can't really uglify css since you have to have the css rules written in something the browser understands and your class names need to match.

the optimization pass is supposed to merge rules together if there have the same selector and to stripe any unecessary chars. you get your space savings from that. after that, there's gzip for more size reduction when it's transferred.